pkgs/development/libraries/lib2geom/default.nix

```diff
@@ -15,27 +15,17 @@
 
 stdenv.mkDerivation rec {
   pname = "lib2geom";
-  version = "1.2.2";
+  version = "1.3";
 
   outputs = [ "out" "dev" ];
 
   src = fetchFromGitLab {
     owner = "inkscape";
     repo = "lib2geom";
     rev = "refs/tags/${version}";
-    sha256 = "sha256-xkUxcAk8KJkL482R7pvgmCT+5I8aUMm/q25pvK3ZPuY=";
+    sha256 = "sha256-llUpW8VRBD8RKaGfyedzsMbLRb8DIo0ePt6m2T2w7Po=";
   };
 
-  patches = [
-    # Fixed upstream, remove when the new version releases:
-    # https://gitlab.com/inkscape/lib2geom/-/issues/49
-    (fetchpatch {
-      name = "expect-double-eq-in-choose-test.patch";
-      url = "https://gitlab.com/inkscape/lib2geom/-/commit/5b7c75dd3841cb415f163f0a81f556c57d3e0a83.patch";
-      sha256 = "RMgwJkylrGFTTrqBzqs5j2LMSLsHhcE/UT1pKBZnU50=";
-    })
-  ];
-
```

Comment:
Per [upstream recommendation](https://gitlab.com/inkscape/lib2geom/-/issues/63#note_1526186398), please add something like the following under the `doCheck` to make it build on aarch64:

```nix
  # TODO: Update cmake hook to make it simpler to selectively disable cmake tests: #113829
  checkPhase = let
    disabledTests =
      lib.optionals stdenv.isAarch64 [
        # Broken on all platforms, test just accidentally passes on some.
        # https://gitlab.com/inkscape/lib2geom/-/issues/63
        "elliptical-arc-test"
      ];
  in ''
    runHook preCheck
    ctest --output-on-failure -E '^${lib.concatStringsSep "|" disabledTests}$'
    runHook postCheck
  '';
```